### DISCUSSION TITLE
Can/should we have `Discussions` for Hyrule?

### DISCUSSION THREAD

**Discussion by gabriel-francischini** [hylang/hy#2226] [Q&A] [answered]
Since the standard library / hyrule is now separated from the language, should we separate the discussions too? Issues regarding hyrule have already been moved to the new repo, apparently

**Comment by allison-casey** [comment]
That's a great idea! Let me look into this

**Comment by Kodiologist** [comment, marked as answer]
Why make people check both, or have to decide where to post things? The fewer distinct fora, the better, no?

**Reply by gabriel-francischini** [reply]
I just thought that there are some discussions that are specific to hyrule. But really maybe it's better to have only one forum, as there always will be things that are Hy-related that don't fit into only two classifications (e.g. people showcasing their Hy libraries or asking for advice to how integrate Hy with their current tools).

**Reply by Kodiologist** [reply]
Yeah, the presumption has generally been that this board is for anything Hy-related, including Hyrule, not just the `hylang/hy` repository.
